### ISSUE TITLE
Hopefully all things Nike asked for this weekend (ending 2 December 2018)

### ISSUE THREAD

**Issue by ndattani** [HPQC-LABS/Pegasus#15]
Referring to version of: https://github.com/HPQC-LABS/Pegasus/blob/ec1fdcb07fd0f5a9c01faf32e7532b45333d1bdb/pegasus.pdf

- [ ] 1) All figures should be **cropped** and **colored** according to this PDF, except perhaps 2b and 2c and the top panels of Fig 3 can use a brighter blue (maybe just [0 0 1]).
- [x] 2a) If Eq. 10 is wrong, fix it, and the description in the sentence before it. 
- [x] 2b) If Eq. 10 is correct, fix Eqs. 9 and 5.
- [x] 3) "compressed"  "one" figure should be in an inset to Fig 2b. This is how to do it: https://tex.stackexchange.com/a/120446/14527
- [ ] 4) Inset to show degree of graph is 15
- [ ] 5) All 3 of these issues: https://github.com/HPQC-LABS/Pegasus/issues/14
- [ ] 6) Other changes from emails and other issues.
- [ ] 7) Now that we know Pegasus a bit better, it might be worth it to see what it would look like if the "+4" edges in each K4,4+4, are grey (or blue) rather than black. Maybe a darker grey than the chimera.... but there's also advantages of making it the same as Chimera.

![selection_999 670](https://user-images.githubusercontent.com/1163013/49483680-1e7e0b00-f802-11e8-8c6c-02bd66ff66a7.png)

No more further changes that I didn't ask for please. Not for now at least. Let me work on getting this paper to a submission-worthy stage because too much has changed since I last did that.

**Comment by ndattani** [comment]
@szilardszalay , @nchancel , what do you guys think about the "+4" pegasus edges within single cells? Should they still be black? Or should they be blue (meaning ALL chimera edges are grey and ALL added pegasus edges are blue), or grey (meaning everything within a single layer z, is grey, and edges BETWEEN different layers are blue) ?

**Comment by nchancel** [comment]
If we are talking about 2(a) I can see either working, In 2(b) I think they should be a different colour which stands out, and the black works pretty well for that.

**Comment by ndattani** [comment]
Of course this is about 2a, because in 2b, blue is used for something very specific.

**Comment by szilardszalay** [comment]
Black is definitely too strong in fig 2(a). I would 
- use grey in 2(a) (single layer = grey, between layers = blue is a good idea),
- but stick to black in fig 2(b) (it emphasizes the new edges compared to Chimera)
Shall I do it in the next commit?

**Comment by ndattani** [comment]
Yes I have mixed feelings about all this, and I keep alternating about what I think would be best, so that's why I value your opinions here.

On one hand, we wanted to emphasize the new edges in Pegasus, 
but on another hand, it doesn't seem right to have 3 different colors when there should only really be 2 categories: (chimera/pegasus, or inter-layer/intra-layer)

If we color based on chimera/pegasus, then the black edges would be light blue. I think this is okay, except that in Fig 2a and similar figures, we would have 6 colors (red/blue/green/orange/grey/light blue) instead of 5. Even if we made these edges black in Fig 2b, we might have to explain why there's 6 colors instead of 5. But the caption is already quite a fair size.

If we color based on inter-layer/intra-layer, we would color these edges grey in both 2a and 2b, which I think is okay. However in the supplemental material, I like having the black edges for the single cell pegasus figures.

This is something that can be done in the next commit (please make sure to pull from master first, so that you're editing the most recent version of the paper), however I think all three of us wouldn't mind submitting the paper without this change of color (it's a minor change that is not of very high priority.. it can be done at any time in the future and it doesn't change the scientific content of the paper). Higher priority would be to finish the SFR gadgets (derivations, gadget graphs, embeddings.... although embeddings will be taken care of by me and Alex).  Also there's the RBS gadget graphs to make (1 for cubic, 2 for quartic) from the email I sent.
